Tests/SnapshotTests.swift

```diff
@@ -35,6 +35,11 @@ class SnapshotTests: XCTestCase {
   /// a sample JSON file that is visible to this test target.
   func testAllSnapshotsHaveCorrespondingSampleFile() {
     for snapshotURL in Samples.snapshotURLs {
+      // Exclude snapshots of private samples, since those aren't checked in to the repo
+      if snapshotURL.lastPathComponent.contains("Private") {
+        continue
```

Comment:
This fails locally for me any time I move or rename samples in the Private folder, which seems unnecessary considering those aren't checked in to the repo